### DISCUSSION TITLE
TabTabTab only displays the tab list correctly when all tabs are also fully loaded in the browser.

### DISCUSSION THREAD

**Discussion by FreePDF64** [okaryo/TabTabTab#111] [Q&A] [answered]
TabTabTab only displays the tab list correctly when all tabs are also fully loaded in the browser.

Is this behavior only with me in the browser?

**Comment by okaryo** [comment]
Thank you for bringing this to our attention!

I haven't observed this behavior before, but there's a possibility that if the HTML of the target page isn't fully loaded when you open the popup, the list might not display correctly. If possible, could you provide a screenshot or video of this issue? It would help a lot!

**Comment by FreePDF64** [comment]
Hi,
It appears when a Tab is not fully loaded. See video:

https://github.com/okaryo/TabTabTab/assets/38452654/32bf9ec1-3752-4ffd-acfa-681c203c486e

**Reply by okaryo** [reply]
Thank you for the video!
I was able to replicate the issue on my end as well. I've identified the cause and will address it in an upcoming update!

**Comment by okaryo** [comment, marked as answer]
I've made the updates in the [v2.15.2](https://github.com/okaryo/TabTabTab/releases/tag/2.15.2) release, so please give it a try!
If there are any unresolved issues or other problems, please let me know.

I'd also love to hear any feedback or slight discomforts you might feel while using TabTabTab. There might be things I've overlooked, and your insights are invaluable for improvements.
